src/Esfa.Vacancy.Register.Infrastructure/Services/TrainingDetailService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Register.Domain.Entities;
+using Esfa.Vacancy.Register.Domain.Interfaces;
+using SFA.DAS.Apprenticeships.Api.Client;
+
+namespace Esfa.Vacancy.Register.Infrastructure.Services
+{
+    public class TrainingDetailService : ITrainingDetailService
+    {
+        public async Task<Framework> GetFrameworkDetails(int code)
+        {
+            using (var client = new FrameworkCodeClient())
```

Comment:
the base url for the api client should be brought in here (from config)
same for standards api client